build/three.js

```diff
@@ -4482,8 +4482,9 @@
 
 	function setValue4fm( gl, v ) {
 
-		gl.uniformMatrix4fv( this.addr, false, v.elements || v );
-
+		if ( v.elements === undefined ) { gl.uniformMatrix4fv( this.addr, false, v ); }
+		else { gl.uniformMatrix4fv( this.addr, false, new Float32Array(v.elements) ); }
```

Comment:
So you're proposing creating a Float32Array every time an uniform gets uploaded?